src/Confix.Tool/src/Confix.Tool/Pipelines/Encryption/OutputFileArgument.cs

```diff
@@ -0,0 +1,19 @@
+using System.CommandLine;
+
+namespace Confix.Tool.Pipelines.Encryption;
+
+internal sealed class OutputFileArgument : Argument<FileInfo?>
+{
+    public static OutputFileArgument Instance { get; } = new();
+
+    private OutputFileArgument()
+        : base("out-file")
+    {
+        Arity = ArgumentArity.ZeroOrOne;
+        Description = """
+            The file to write the encrypted or decrypted data to. 
```

Comment:
How does that look in the output?

Reply:
![image](https://github.com/SwissLife-OSS/Confix/assets/20257550/a7a8f8d3-97ce-4356-ad24-cc3fd5e57bae)